### PR TITLE
Refs Task #397809 unit test show detail Admin Payload History

### PR DESCRIPTION
**Pull request by quanghung97**
## Related Tickets

- [#397809](https://dev.sun-asterisk.com/issues/397809)

## What's this PR do ?

- [x] Unit test show detail Admin Payload History

## Performance

- [ ] Resolved n + 1 query
- [ ] Run explain query already
- [ ] Time run rake task : 1000 ms

## Checklist

- [x] It was tested in local success?
- [x] Fill link PR into ticket and the opposite
- [x] Note purpose/reason, solution, scope of influence into ticket

## ENV note

```
```

## Notes
*(Other notes)*

### CODE REVIEW

**Comment by framgiaciv3** [comment]
coverage</br> **1. Result build** </br><table><tr><th>Style</th><th>Status</th><th>Ignored</th></tr><tr><td>cp__env_example__env._start_run_.3</td><td>success</td><td></td></tr><tr><td>phpcs</td><td>success</td><td></td></tr><tr><td>php_artisan_config_c._start_run_.11</td><td>success</td><td></td></tr><tr><td>composer_dump-autolo._start_run_.17</td><td>success</td><td></td></tr><tr><td>yarn_dev._start_run_.9</td><td>success</td><td></td></tr><tr><td>composer_install._start_run_.5</td><td>success</td><td></td></tr><tr><td>yarn_install._start_run_.7</td><td>success</td><td></td></tr><tr><td>framgia-ci_test-conn._start_run_.15</td><td>success</td><td></td></tr><tr><td>php_artisan_migrate_._start_run_.19</td><td>success</td><td></td></tr><tr><td>php_artisan_key_gene._start_run_.13</td><td>success</td><td></td></tr><tr><td>phpunit</td><td>success</td><td></td></tr></table>

File: app/Http/Controllers/Admin/PayloadHistoryController.php [hunk 1]
```diff
@@ -51,7 +51,7 @@ public function show(Request $request, $id)
 
         $this->authorize('show', $payloadHistory);
 
-        $messageHistories = $this->messageHistoryRepository->adminGetAllAndSearch($id, $keyword);
```
Comment (ghost) [inline app/Http/Controllers/Admin/PayloadHistoryController.php:54]: pass dk CI cũng hay hè :D

**Review by tamtt-1031 — APPROVED** [review]

File: tests/Feature/PayloadHistoryControllerTest.php [hunk 2]
```diff
@@ -65,6 +65,53 @@ public function testShowDetailPayloadHistoryAdminUserFeature()
         $response->assertViewHas('messageHistories');
     }
 
+    /**
+     * test Feature show admin payload history.
+     *
+     * @return void
+     */
+    public function testShowAdminDetailPayloadHistoryFeature()
```
Comment (ghost) [inline tests/Feature/PayloadHistoryControllerTest.php]: tạo 1 folder admin trong test folder, rồi viết trong đó đi chú, đừng viết chung tn

**Review by nguyenthinhhn — APPROVED** [review]

File: app/Http/Controllers/Admin/PayloadHistoryController.php (hunk 1 above)
Comment (quanghung97) [reply app/Http/Controllers/Admin/PayloadHistoryController.php:54]: cái cũ Thịnh nó chưa viết unitest cho cái hàm này.

**Review by ghost — APPROVED** [review]

**Comment by framgiaciv3** [comment]
coverage</br> **1. Result build** </br><table><tr><th>Style</th><th>Status</th><th>Ignored</th></tr><tr><td>framgia-ci_test-conn._start_run_.15</td><td>success</td><td></td></tr><tr><td>cp__env_example__env._start_run_.3</td><td>success</td><td></td></tr><tr><td>yarn_install._start_run_.7</td><td>success</td><td></td></tr><tr><td>yarn_dev._start_run_.9</td><td>success</td><td></td></tr><tr><td>php_artisan_migrate_._start_run_.19</td><td>success</td><td></td></tr><tr><td>phpunit</td><td>success</td><td></td></tr><tr><td>php_artisan_key_gene._start_run_.13</td><td>success</td><td></td></tr><tr><td>composer_install._start_run_.5</td><td>success</td><td></td></tr><tr><td>composer_dump-autolo._start_run_.17</td><td>success</td><td></td></tr><tr><td>phpcs</td><td>success</td><td></td></tr><tr><td>php_artisan_config_c._start_run_.11</td><td>success</td><td></td></tr></table>

**Review by ghost — APPROVED** [review]

**Comment by phucqd-1305** [comment]
OK

**Pull request merged**